### ISSUE TITLE
Hide any attribute not working 1.21

### ISSUE THREAD

**Issue by erneto13** [emanondev/ItemEdit#25]
# ItemEdit Hide

The /ie hide <item-flags> command does not work in the latest version of MC. It also doesn't work with the /ie hideall command.

![image](https://github.com/emanondev/ItemEdit/assets/128459472/c8ff72da-4aaf-499d-bf85-658bae47c968)

**Comment by emanondev** [comment]
several ppl reported this issue however i was unable to replicate it under this test environment:
```
[17:53:42] [Server thread/INFO]: Server: 4226-Spigot-146439e-2889b3a (MC: 1.21)
[17:53:42] [Server thread/INFO]: ItemEdit: 3.4.2 Storage: YAML
[17:53:42] [Server thread/INFO]: ItemTag: 3.4.2
[17:53:42] [Server thread/INFO]: PAPI: 2.11.6
[17:53:42] [Server thread/INFO]: NBTAPI: Nope
[17:53:42] [Server thread/INFO]: Vault: Nope
[17:53:42] [Server thread/INFO]: MythicMobs: Nope
[17:53:42] [Server thread/INFO]: ShopGuiPlus: Nope
[17:53:42] [Server thread/INFO]: Vanish: Nope
```

can you do /itemeditinfo and paste here the result so i can check if there are any relevant differences of server version/plugin version?

**Comment by erneto13** [comment]
sure
![image](https://github.com/emanondev/ItemEdit/assets/128459472/4caf1945-a321-4980-a39a-17814f296eae)

**Comment by erneto13** [comment]
have you managed to find out anything more about the subject?

**Comment by emanondev** [comment]
yes it's a bug on PaperSpigot from 1.20.6 and later versions, Purpur which is a fork of Paper has the same issue of course
that only happens for default attributes, custom ones are hidden by the flag as it is supposed to

unluckly no fix is planned according to paper developer(s)
https://github.com/PaperMC/Paper/issues/10693

the only walk around right now is to add custom attrubutes matching the default ones and have them hidden by the flag

**Comment by emanondev** [comment]
i found a walkaround setting custom attributes equals to default 

for now that's the best compromise we can have
